From 5476d2cf4e659fd54513bbe0634b7865a0be97e8 Mon Sep 17 00:00:00 2001
From: Mervin Fansler <mmfansler@gmail.com>
Date: Wed, 14 Jun 2023 16:46:37 -0400
Subject: [PATCH] skip tests on ci

---
 tests/testthat/test-multi.R | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/tests/testthat/test-multi.R b/tests/testthat/test-multi.R
index 816b4c3..4fd8d33 100644
--- a/tests/testthat/test-multi.R
+++ b/tests/testthat/test-multi.R
@@ -138,6 +138,7 @@ test_that("callback protection", {
 })
 
 test_that("total_con and multi_fdset", {
+  skip_on_ci()
   skip_on_os("solaris")
   skip_if_not(strsplit(curl_version()$version, "-")[[1]][1] >= as.numeric_version("7.30"),
               "libcurl does not support host_connections")
@@ -155,6 +156,7 @@ test_that("total_con and multi_fdset", {
 })
 
 test_that("host_con and multi_fdset", {
+  skip_on_ci()
   skip_on_os("solaris")
   skip_if_not(strsplit(curl_version()$version, "-")[[1]][1] >= as.numeric_version("7.30"),
               "libcurl does not support host_connections")
