From 715f86ca0a0fad2cb7394dd0fce8fa8accb4da4c Mon Sep 17 00:00:00 2001
From: Mervin Fansler <mmfansler@gmail.com>
Date: Wed, 14 Jun 2023 22:56:33 -0400
Subject: [PATCH] skip if no webutils

---
 tests/testthat/test-echo.R | 1 +
 1 file changed, 1 insertion(+)

diff --git a/tests/testthat/test-echo.R b/tests/testthat/test-echo.R
index 3b2976b..be96461 100644
--- a/tests/testthat/test-echo.R
+++ b/tests/testthat/test-echo.R
@@ -2,6 +2,7 @@ context("echo")
 
 test_that("roundtrip large data", {
   skip_if_not_installed('httpuv')
+  skip_if_not_installed('webutils')
   bindata <- serialize(rnorm(1e5), NULL)
   input_url <- 'https://fakeserver.org:99/my/endpoint'
   handle <- curl::new_handle(url = input_url)
